### DISCUSSION TITLE
New Message - Active Window for Output

### DISCUSSION THREAD

**Discussion by rawkode** [YaLTeR/niri#622] [Ideas]
I'm writing a Waybar module to display the window title. I would like each output to display the active window for that output.

I don't believe this is possible with the current messages available.

I know we have `focused-window`, but that doesn't allow for filtering by output and technically there can only be one focused window across all outputs.

Is there any way to get the information I rrequire?

**Comment by YaLTeR** [comment]
I don't think there is atm.
